src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -339,27 +373,11 @@ public function moveNodeImmediately($srcAbsPath, $dstAbsPath)
     /**
      * {@inheritDoc}
      */
-    public function reorderNodes($absPath, $reorders)
```

Comment:
i think it was renamed to reorderChildren. we might need to capture that in order to invalidate caches. actually it would probably be enough to invalidate the cache for just the one node that is passed in, as reordering has no other effects than on the parent node of the reordered nodes.